site/src/pages/components/menu.explainer.mdx

```diff
@@ -35,51 +35,50 @@ Use cases:
-    * If type is checkbox or radio, it is automatically checkable
-* Add a **checkable** attribute on the `<menuitem>`
-    * menuitem.checked IDL property to check if the item is checked
-    * Add **name** attribute to group items together if we only want one element to checked at a time (for type radio)
+* Events:
+    * "toggle" on popover will be fired.
+    * "change" will be fired when the `<menuitem>` is checkable, and its state changes
+* Checkability:
+    * Supports "checkable" configuration and state, by wrapping `<menuitem>`s in a `<fieldset>` with
+      a new `checkable` attribute; see [Grouping menuitems together](#grouping-menuitems-together).
+    * Supports the :checked CSS pseudo-class and ::checkmark CSS pseudo-element
+
+**Question: Should `<menuitem>`s be represented by a single element, or multiple elements?**
+
+As stated, our proposal involves introducing a single `<menuitem>` element to represent menu
+buttons, some of which can be checkable. Instead of representing both non-checkable & checkable
+state-bearing menu items with the single `<menuitem>` element, we could also consider:
+
+* Add a `type` attribute: `<menuitem>`, `<menuitem type=checkbox>`, `<menuitem type=radio>`.
+    * If `type` is `checkbox` or `radio`, it is automatically checkable
+* Add a `checkable` IDL attribute on `<menuitem>`
+    * This IDL property could be used to see if the item is checked
+    * Use a `name` attribute to group exclusive menu items together (this only applies for "radio"
+      menu items):
         * `<menuitem checkable name=foo>`
         * `<menuitem checkable name=foo>`
-* Add a **checkable** attribute on the grouping element (fieldset, menulist)
-    * Checkable attribute can be multiple (for checkbox) or single (for radio)
-    * `<fieldset checkable=multiple><menuitem>`
-    * `<fieldset checkable=single><menuitem>`
+* Add a `checkable` IDL attribute on the *grouping* element (`<fieldset>`, `<menulist>`)
+    * This attribute can have the values `single` (for radios) or `multiple` (for checkbox)
 
-**Question: Should we allow `<menuitemcheckbox>`, `<menuitemradio>` in `<menubar>`?**
+**Question: Should we allow radio and checkbox menu items as direct children of a `<menubar>`?**
 
-Currently, we don’t see a need for this. All menuitems inside a menubar should open a menulist. We are open to removing this restriction if there are good use cases. For example, for a toolbar pattern (which this proposal isn’t covering), a `<menuitem>` with the text "Bold" should mark that element as checked.
+Currently, we don’t see a need for this. All menuitems inside a menubar should perform some action
+like opening a sub `<menulist>`, however we will reconsider this restriction if good use cases
+arise.
 
+For example, for the toolbar pattern (which this proposal isn't yet covering), a `<menuitem>` with
+the text "Bold" should be checkable, to support a text editor.
```

Comment:
use this is you like... but figured this could be revised to remove the reference to `toolbar` since it was resolved to not cover that use case with this proposal - but the example is still a valid and easy to understand use case for checkable menuitems.

```suggestion
For example, a text editor might need to provide a `<menulist>` to provide basic font style adjustments. A group of checkable `<menuitem>` elements to mark content as "Bold", "Italic" or "Underlined", could be provided.  The checked state representing whether the text is currently "Bold" (checked) or not.
```